.github/workflows/clean-stale-branches.yaml

```diff
@@ -0,0 +1,23 @@
+name: Cleanup Stale Branches
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # This schedule runs the workflow at midnight every day
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
Do we need this?

Reply:
testing, to trigger the workflow and understand what is happening 

Dry run is set to true